src/__tests__/SelectPanel.tsx

```diff
@@ -0,0 +1,65 @@
+import {cleanup, render as HTMLRender} from '@testing-library/react'
+import 'babel-polyfill'
+import {axe, toHaveNoViolations} from 'jest-axe'
+import React from 'react'
+import theme from '../theme'
+import {SelectPanel} from '../SelectPanel'
+import {COMMON} from '../constants'
+import {behavesAsComponent, checkExports} from '../utils/testing'
+import {BaseStyles, ThemeProvider} from '..'
```

Comment:
If you run `yarn run dist` does it warn about a circular reference? If so, I’d suggest `import`’ing from a location other than `..`.